.github/workflows/ci.yml

```diff
@@ -16,11 +16,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest]
         node_version: [12, 14, 16]
-        include:
-          - os: windows-latest
-            node_version: 16
```

Comment:
Initially I was worried that adding too many Windows images would slow tests down. But on a whim I just tested Windows in all versions and it seemed to work. Probably for users’ sakes we should just test all Node versions in all OSs.

Reply:
I think you need multiple entries for `include` like I did here:

https://github.com/snowpackjs/prettier-plugin-astro/blob/f39ebbd060f44aeac92cc1d073067ba13cdb1f03/.github/workflows/ci.yml#L17-L20

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-including-new-combinations